src/components/Person/Person.jsx

```diff
@@ -1 +1,11 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
```

Comment:
Use destructuring for `person`

Reply:
I don't have to do this beacuse of that:
![image](https://github.com/user-attachments/assets/fb22bc4b-8bc3-40de-9287-87d065d0b6ed)